src/status_im/multiaccounts/core.cljs

```diff
@@ -103,6 +103,12 @@
              :preview-privacy? (boolean private?)
              {})))
 
+(fx/defn switch-webview-permission-requests?
+  [{:keys [db] :as cofx} enabled?]
+  (multiaccounts.update/multiaccount-update cofx
```

Comment:
can we move `cofx` to the next line, that would help to keep it in 80 chars range